### ISSUE TITLE
`value-keyword-case` does not ignore `.tsx` files

### ISSUE THREAD

**Issue by JounQin** [stylelint/stylelint#5552]
> Clearly describe the bug

As title

> Which rule, if any, is the bug related to?

`value-keyword-case`

> What code is needed to reproduce the bug?

```tsx
<div
  className="card-item"
  style={{
    backgroundColor,
    backgroundPosition: hoverBgPosition,
    backgroundSize: hoverBgSize,
  }}
></div>
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "value-keyword-case": "lower"
  }
}
```

> Which version of stylelint are you using?

e.g. `13.13.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint . --cache`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
src/pages/Products/product-card.tsx
 49:37  ✖  Expected "backgroundColor" to be "backgroundcolor"   value-keyword-case
 50:42  ✖  Expected "hoverBgPosition" to be "hoverbgposition"   value-keyword-case
 51:38  ✖  Expected "hoverBgSize" to be "hoverbgsize"           value-keyword-case
```

**Comment by jeddy3** [comment]
@JounQin Thanks for the report and for using the template.

You can use a glob to target the specific files you want to lint, e.g.:

```
stylelint "**/*.css"
```

---

You've likely uncovered a bug with the [upstream parser](https://github.com/stylelint/postcss-css-in-js), but that is [being removed in the upcoming release](https://github.com/stylelint/stylelint/issues/5205).
